tidalapi/workers.py

```diff
@@ -0,0 +1,56 @@
+from concurrent.futures import ThreadPoolExecutor
+from typing import Callable
+
```

Comment:
```suggestion
import logging

log = logging.getLogger(__name__)

```

See https://github.com/EbbLabs/python-tidal/pull/351/files#r2241068508